sbi/analysis/test_utils.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+import pandas as pd
+
+
+def PP_vals(RV_samples, alphas):
```

Comment:
you should avoid capitals in function name and variable names (cf pep8 conventions)
```suggestion
def pp_vals(rv_samples, alphas):
```